ui/src/main/js/application/Navigation.js

```diff
@@ -36,67 +61,132 @@ const Navigation = ({ confirmLogoutPressed, cancelLogout, globalDescriptorMap })
+    //                         Channels
+    //                     </li>
+    //                 )}
+    //                 {doesDescriptorExist(globalDescriptorMap, AZURE_BOARDS_INFO.key) && createStaticNavItem(channelUri, AZURE_BOARDS_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, EMAIL_INFO.key) && createStaticNavItem(channelUri, EMAIL_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, JIRA_CLOUD_INFO.key) && createStaticNavItem(channelUri, JIRA_CLOUD_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, JIRA_SERVER_INFO.key) && createStaticNavItem(channelUri, JIRA_SERVER_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, MSTEAMS_INFO.key) && createStaticNavItem(channelUri, MSTEAMS_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, SLACK_INFO.key) && createStaticNavItem(channelUri, SLACK_INFO)}
+    //                 <li className="navHeader">
+    //                     Jobs
+    //                 </li>
+    //                 <li>
+    //                     <NavLink to={DISTRIBUTION_URLS.distributionTableUrl} activeClassName="activeNav">
+    //                         {DISTRIBUTION_INFO.label}
+    //                     </NavLink>
+    //                 </li>
+    //                 <li className="divider" />
+    //                 {doesDescriptorExist(globalDescriptorMap, AUDIT_INFO.key) && createStaticNavItem(componentUri, AUDIT_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, AUTHENTICATION_INFO.key) && createStaticNavItem(componentUri, AUTHENTICATION_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, CERTIFICATE_INFO.key) && createStaticNavItem(componentUri, CERTIFICATE_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, SCHEDULING_INFO.key) && createStaticNavItem(componentUri, SCHEDULING_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, SETTINGS_INFO.key) && createStaticNavItem(componentUri, SETTINGS_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, TASK_MANAGEMENT_INFO.key) && createStaticNavItem(componentUri, TASK_MANAGEMENT_INFO)}
+    //                 {doesDescriptorExist(globalDescriptorMap, USER_MANAGEMENT_INFO.key) && createStaticNavItem(componentUri, USER_MANAGEMENT_INFO)}
+    //                 <li className="logoutLink">
+    //                     <a
+    //                         role="button"
+    //                         tabIndex={0}
+    //                         onClick={(evt) => {
+    //                             evt.preventDefault();
+    //                             confirmLogoutPressed();
+    //                         }}
+    //                     >
+    //                         Logout
+    //                     </a>
+    //                 </li>
+    //             </ul>
+    //         </div>
+    //     </div>
+    // );
```

Comment:
Will these lines be removed prior to merge?

Reply:
Yep! Should be removed now 👍